android/build.gradle

```diff
@@ -34,5 +34,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
```

Comment:
@LydGol90 is everything fine on the old version of Android after this change?

Reply:
hmm I may need to check, the min sdk version I am using is 18 but I see that yours is 16

Reply:
@LydGol90 any update on checking support for 16 with your proposed change? Would be good to get the deprecation warning gone.